sdk/documenttranslation/azure-ai-documenttranslation/azure/ai/documenttranslation/_client.py

```diff
@@ -152,7 +162,8 @@ def get_supported_glossary_formats(self, **kwargs):
         :rtype: List[FileFormat]
         """
 
-        return self._client.document_translation.get_glossary_formats(**kwargs)
+        glossary_formats = self._client.document_translation.get_glossary_formats(**kwargs)
+        return FileFormat._from_generated_list(glossary_formats)
```

Comment:
i used to do it inline, but i thought it's much cleaner if i created "_from_generated_list()" to process lists
instead of having to same code for the same object over and over